frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -170,11 +160,38 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
   }, [namespace, pvcName]);
 
   const [data, loaded, loadError] = useK8sWatchResource<PersistentVolumeClaimKind[]>(resourceWatch);
+  const scList = scObjListLoaded ? (scObjList.items as StorageClassResourceKind[]) : [];
+  const snapshotClassFilter = useCallback(
+    (snapshotClass: VolumeSnapshotClassKind) => scObj?.provisioner.includes(snapshotClass?.driver),
+    [scObj],
+  );
+  const filterWrapper = (storageClassObj) => {
+    return (snapshotClass: VolumeSnapshotClassKind) => {
+      return storageClassObj?.provisioner.includes(snapshotClass?.driver);
+    };
+  };
+  const vscList = vscLoaded ? (vscObj.items as VolumeSnapshotClassKind[]) : [];
+  const getDefaultItem = useCallback(
+    (snapFilter) => {
+      const filteredVSC: VolumeSnapshotClassKind[] = vscList.filter(snapFilter);
```

Comment:
no need to specify the type again
```suggestion
      const filteredVSC = vscList.filter(snapFilter);
```

Reply:
removed